src/fides/api/models/privacy_request/privacy_request.py

```diff
@@ -893,9 +893,12 @@ def trigger_pre_approval_webhook(
         To respond, the service should send a request to one of the reply-to URLs with the reply-to-token.
         """
         # temp fix for circular dependency
-        from fides.api.service.connectors import HTTPSConnector, get_connector
+        from fides.api.service.connectors import (
+            HTTPSConnector,
+            get_connector,
+        )
 
-        https_connector: HTTPSConnector = get_connector(webhook.connection_config)  # type: ignore
+        https_connector: Union[HTTPSConnector | HTTPSOAuth2Connector] = get_connector(webhook.connection_config)  # type: ignore
```

Comment:
Python 3.9 will complain about the use of `|`, so let's go with this
```suggestion
        https_connector: Union[HTTPSConnector, HTTPSOAuth2Connector] = get_connector(webhook.connection_config)  # type: ignore
```

Reply:
It also looks like `HTTPSOAuth2Connector` isn't defined?

Reply:
Yeah, looks like I missed a few hunks / lines when I pushed it up for draft, I will push them up along with the OAuth grant type cleanup